kbs/docker/Dockerfile

```diff
@@ -41,7 +41,7 @@ COPY . .
 RUN cd kbs && make AS_FEATURE=coco-as-builtin ALIYUN=${ALIYUN} ARCH=${ARCH} && \
     make ARCH=${ARCH} install-kbs
 
-FROM ubuntu:22.04
+FROM ubuntu:24.04
```

Comment:
In this file for sure additional change needs to be made - include gpg package in apt-get -> without gpg building will fail with `14.17 /bin/sh: 1: gpg: not found`

Reply:
`apt-key: not found` should be handled, too.

Reply:
This PR should be rebased and tested after https://github.com/confidential-containers/trustee/pull/923 is merged.